Sources/Authenticator/Views/ContinueSignInWithMFASelectionView.swift

```diff
@@ -12,7 +12,11 @@ import SwiftUI
 public struct ContinueSignInWithMFASelectionView<Header: View,
                                                  Footer: View>: View {
     @Environment(\.authenticatorState) private var authenticatorState
-    @ObservedObject private var state: ConfirmSignInWithCodeState
+    @ObservedObject private var state: ContinueSignInWithMFASelectionState
+
+    /// The MFA selection  provided by the user
+    @State private var selectedMFAType: MFAType?
```

Comment:
Can this be a `@Published` property in the observed `ContinueSignInWithMFASelectionState` object instead, to be consistent with all the other views/states?